web/views/password_reset_view.ex

```diff
@@ -1,9 +1,11 @@
 defmodule CodeCorps.PasswordResetView do
   use CodeCorps.Web, :view
 
-  def render("show.json", %{token: token}) do
+  def render("show.json", %{email: email, token: token, user_id: user_id}) do
     %{
-      token: token
+      email: email,
+      token: token,
```

Comment:
Not seeing the token/user_id being consumed on the [frontend](https://github.com/code-corps/code-corps-ember/blob/84d91618ccb464b112555298600b366aeb53dc7f/app/routes/password/reset.js#L30).